fastapi/openapi/utils.py

```diff
@@ -421,6 +422,7 @@ def get_openapi(
             )
             if result:
                 path, security_schemes, path_definitions = result
+                route.path_format = prefix + route.path_format
```

Comment:
I suggest using the f-string feature to concatenate strings. Do not create new commits for requested changes. Just amend commits that are the changes requested.

Reply:
Go ahead :) https://stackoverflow.com/questions/7947322/preferred-github-workflow-for-updating-a-pull-request-after-code-review

Reply:
What changes if I use f-string?

Reply:
```py
route.path_format = f'{prefix}{route.path_format'
```

Reply:
Yes, but is there any reason to use this instead of +?

Reply:
Yeah. It makes it easy to read.